.changeset/friendly-monkeys-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-graph': patch
+---
+
+Add tokens for '@' and '$'.
```

Comment:
We can leave this blank. Don't think a changeset message is required for this small of a change.